client/src/components/Home2nd.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from "react";
+import styled from "styled-components";
+import Header from "./Header";
+import CitySlider from "./CitySlider";
+
+const LogoWrapper = styled.div`
+  text-align: center;
+`;
+
+const Logo = styled.img`
```

Comment:
I like the use of the string partials 👍 
I personally haven't used them on a large project but I would be curious to see how they scale when using inheritance.